mis_builder/views/mis_report_instance.xml

```diff
@@ -231,6 +231,7 @@
                         <group name="relative" attrs="{'invisible': [('mode', '!=', 'relative')]}" colspan="4">
                             <group>
                                 <field name="type" attrs="{'required': [('mode', '=', 'relative')]}"/>
+                                <field name="is_ytd" force_save="1"/>
```

Comment:
@MaxyMoos is force_save necessary? Why?

Reply:
Well, to be honest, I can't remember now what problem I was trying to solve by using this...
But trying it out quickly today I can't seem to find any issues arising without having the `force_save` attribute, so I'll get rid of it.